src/features/events/components/EventOverviewCard/EventParticipantsCard.tsx

```diff
@@ -0,0 +1,221 @@
+
+                        if (val == '') {
+                          setNewReqParticipants(null);
+                          return;
+                        }
+
+                        const intVal = parseInt(val);
+                        if (!isNaN(intVal) && intVal.toString() == val) {
+                          setNewReqParticipants(intVal);
+                        }
+                      }}
+                      onKeyDown={(ev) => {
+                        if (ev.key === 'Enter') {
+                          setAnchorEl(null);
+                          if (newReqParticipants != null) {
+                            model.setReqParticipants(newReqParticipants);
+                          }
+                        } else if (ev.key === 'Escape') {
+                          setAnchorEl(null);
+                          setNewReqParticipants(reqParticipants);
+                        }
+                      }}
+                      value={
+                        newReqParticipants === null ? '' : newReqParticipants
+                      }
+                      variant="outlined"
+                    />
+                  </Box>
+                </Paper>
+              </ClickAwayListener>
+            </Popper>
+          </Box>
+        }
+      >
+        <Divider />
+        <Box
+          display="flex"
+          flexDirection="column"
+          justifyContent="space-between"
+        >
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginY={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
+              component="h6"
+              variant="subtitle1"
+            >
+              {messages.eventParticipantsCard.pending()}
+            </Typography>
+            <Typography>
+              {Math.max(reqParticipants - availParticipants, 0)}
+            </Typography>
+          </Box>
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginBottom={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
+              component="h6"
+              variant="subtitle1"
+            >
+              {messages.eventParticipantsCard.booked()}
+            </Typography>
+            <Typography>{`${availParticipants}/${availParticipants}`}</Typography>
+          </Box>
+          <Box
+            alignItems="center"
+            display="flex"
+            justifyContent="space-between"
+            marginBottom={1}
+          >
+            <Typography
+              color={theme.palette.text.secondary}
+              component="h6"
+              variant="subtitle1"
+            >
+              {messages.eventParticipantsCard.contact()}
+            </Typography>
+            {eventData.contact && (
+              <ZUIPerson
```

Comment:
When playing with it I felt like I expected to be able to interact with the contact person in some way - that it would be a ZUIPersonLink (that has a ZUIPersonHoverCard built into it). Is it designed to intentionally not be interactive @richardolsson @Bagera @yusf ?

Reply:
I agree that interaction is expected in this case. I will add it for now, and then we can always revert that change if deemed so.